lib/babel-plugin-inline-svg.js

```diff
@@ -46,6 +47,7 @@ module.exports = ({ types }) => ({
         const input = readFileSync(svgPath, "utf8");
 
         const output = compose(
+          !exportDataURI ? noop : createDataURI,
```

Comment:
This might read better actually:

```suggestion
          exportDataURI ? createDataURI: noop,
```

Reply:
Yes, I have chosen the pattern of the right side of the question mark (noop : function), but since this is an afirmative parameter (contrasting with the other parameters, that disable something), we'll have some pattern breaking... 

Done!